### ISSUE TITLE
[DatePicker] Accessibility issue for missing form label

### ISSUE THREAD

**Issue by georgi-sla** [telerik/kendo-angular#4124]
**Describe the bug**
Currently, the DatePicker component shows an accessibility error for missing form label in the WAVE accessibility evaluation tool. 

**To Reproduce**
1. Create a DatePicker component
2. Associate it with the Label component (`kendo-label`). 
3. Test it in the WAVE accessibility evaluation tool.

Please, check out the following StackBlitz demo: https://stackblitz.com/edit/angular-p7re2y-6u4y86

**Expected behavior**
The DatePicker component should not throw any accessibility issues in the WAVE accessibility evaluation tool. 

**Additional context**
For more details, please check out the private support ticket with ID: 1626531

**Comment by stnikolova** [comment]
Fixed in **v14.1.0**.
